### PR TITLE
Syndicate are no longer gullible about collaborators

### PR DESCRIPTION
**Pull request by Kyep**
🆑 Kyep
rscdel: The syndicate are no longer so gullible as to believe that 100% of crew who are 'opposed' to NT, and 50% of crew who are 'Skeptical' of NT, would be willing to help their agents blow up the Cyberiad. Thus, they will no longer label such people 'potential collaborators'.
/🆑

**SQL to drop the col:** ALTER TABLE SS13_characters DROP COLUMN nanotrasen_relation;

### CODE REVIEW

**Comment by gt33** [comment]
> PDA's collab codewords
> gets sent with large bomb to toxins testing area
> yep we can collab

**Comment by Necaladun** [comment]
Causes way more confusion to newbies than it is ever usefull...which is basically never. Especially when the HoS or Captain is your damn collaborator. 

Haven't seen anyone come up with a better system in ~3 years, best to just remove.

**Comment by tylerh95** [comment]
@gt33 was dat you? im so sorry :<

Anyways, +1 on PR, many of the people set to being opposed and skeptical aren't even willing to help traitors.

**Comment by SheepFighter** [comment]
Oh heck, is *that* what that does?

Excuse me while I die of laughter...

**Comment by mrturkeytoe** [comment]
Why not have like, a checkbox in the preference options for whether you want to be listed as a potential collaborator if you roll traitor? So not everyone has to take the risk of being outed but have the greater possibilities for cooperation if they so choose.

**Comment by tylerh95** [comment]
@mrturkeytoe 

I actually like the concept of the collaboration system, it blurs the lines between traitors and crew. However, in the current practice, it's useless. Perhaps if setting your NT alignment to opposed brought up a dialog box informing the person of what that entails would work. Admins would also perhaps have to enforce this slightly higher level of RP.

For example, if a traitor calls upon someone who is opposed to NT to do something like fetch them a chemical or whatever, and that person just turns around and informs security, they should get spoken to or such. I don't know.

**Comment by Aurorablade** [comment]
You could just add a line somewhere to explain it instead of removing it. It has led to good rp in the past. Plus its why syndicate have code words.

**Comment by mrturkeytoe** [comment]
Well that's why I suggested it be optional, so people'd be taking the risk of being hunted down.

**Comment by jughu** [comment]
People choose skeptical yet they dont rp it 🤔

**Comment by theColdflame** [comment]
being skeptical about nt doesn't imply you like the syndicate any better or are willing to help a literal terrorist

**Comment by FalseIncarnate** [comment]
A large part of the issue is also the (valid) fear of self-antagging punishment for collaborators. Obviously this is a case of using your best judgement, but some people are afraid that knowingly aiding an antag is gonna get them in trouble.

I'll admit, I've been selected as someone's collaborator and reported them to security after they tried getting me to help them... though they were WAY too obvious about it (plus got mad at me for not knowing codewords). Frankly the AI could have heard their admission of being a traitor it was so obvious.

I think maybe instead of removing this outright, we could instead try giving the collaborators codewords (perhaps one of the 3 possible sets each, so one person can't turn around and tell sec all of them at start), prevent protected (traitor-proof) jobs from being collaborators, and give the collaborator a vague message when selected like "You suddenly remember being slipped a note this morning saying an agent may contact you to help strike a blow at your common enemy." This would make collaborators actually be aware they are collaborators, but not give them a single traitor's name (so they can't run and report them to sec before being approached).

**Comment by tigercat2000** [comment]
This makes the NT Status preference entirely pointless, you can't even see it from any other place. Remove it entirely if you're going to do this.

I don't think you should remove this, though. The idea of them is for agents to have possibilities for people who may help them- Yes, it could be a bit more clear what exactly they are, given the fact that they are rarely if ever antagonists, but it's not entirely useless.

**Comment by Aurorablade** [comment]
That's the point of code words. If they don't reply to codewords with codewords then they are not a syndicate
 Also what false and tiger said. This is better not being removed.

**Comment by tigercat2000** [comment]
@Aurorablade You're correct, but it also doesn't even matter if they are syndicate, the status simply marks someone who may be willing to help you- antag or not. Moderate aid to antagonists is not against our rules, nothing says you can't steal (non-high risk) items for them, etc.

**Comment by taukausanake** [comment]
I personally don't like trying for traitor or antag role (mainly because I'm not robust enough) but I would so go for the chance to help someone with their own objectives. Passively of course as can't self-antag even if asked by one but if someone wanted a bomb from Toxins or a chem grenade from Chemistry or Test Lab for chaos I wouldn't mind helping them out granted my character feels that way

**Comment by SamHPurp** [comment]
I dunno, I think I'd still rather this stay in, but with logical improvements such as clarifying the wording and blacklisting mindshielded roles.

**Comment by Lady-Luck** [comment]
I think that potential collaborators should be removed for now, but not the NT Status preference. I think there may be use for the latter in the future, so stripping out the framework is just going to be SQL fuckery.

**Comment by Fox-McCloud** [comment]
As Tigercat said, this removes the last relic of NT position actually having any impact--if this is to be done, then NT relation needs to be stripped out entirely.

**Comment by Lady-Luck** [comment]
Isn't it a bit wasteful to do a bunch of SQL changes for something that might have a use again in the future?

**Comment by taukausanake** [comment]
Isn't it a bit wasteful to remove something that might be refactored in the future?

**Comment by Fox-McCloud** [comment]
Unless something is incredibly tedious to add/remove and has clear definitive direct uses in the future, there's not much of a point to keeping it around.

**Comment by Kyep** [comment]
@Fox-McCloud Wish granted. This PR now removes it completely.

**Comment by jughu** [comment]
Keep it in REEE

File: code/modules/client/preference/preferences_mysql.dm [hunk 1]
```diff
@@ -261,21 +260,19 @@
 		player_alt_titles = params2list(query.item[58])
 		organ_data = params2list(query.item[59])
 		rlimb_data = params2list(query.item[60])
-		nanotrasen_relation = query.item[61]
-		speciesprefs = text2num(query.item[62])
+		speciesprefs = text2num(query.item[61])
```
Comment (Kyep) [inline code/modules/client/preference/preferences_mysql.dm]: Are you sure? If so, why do you think that? Above this, we eliminate nanotrasen_relation from the list. Additionally, there is SQL in the OP to drop the col.

Comment (FalseIncarnate) [inline code/modules/client/preference/preferences_mysql.dm]: Existing database entries aren't going to match the new indexes for species prefs onward unless the entire column gets removed from all records in the database... I suppose that really all depends on how much attention the person making the database changes pays...

Afterall, knee-high socks aren't an acceptable species.

Comment (Kyep) [inline code/modules/client/preference/preferences_mysql.dm]: That is what the SQL in the OP does.

Comment (FalseIncarnate) [inline code/modules/client/preference/preferences_mysql.dm]: I must have missed that bit when i typed the comment initially. Ignore this then.

**Comment by Fraility1** [comment]
I'd rather keep it in and like people have said, find a way to make it work rather then removing it.

It can add some interesting roleplay.
It's not really broke just misused, so rather then remove it clarify it. I for one did not know what it did until i read this thread.

**Comment by tigercat2000** [comment]
@Kyep By the way, one note on this- I dunno if the text is still there, but on Bay, the NT Opposition status is described as "not your character's opinion towards NT but their history and actions with the company". So, "Opposed" means they have already done something nasty towards the company, in theory.

**Comment by taukausanake** [comment]
@tigercat2000 By theory almost everyone would be opposed then, unless you don't take into consideration past antag rounds. And then you have to wonder why NT hired a criminal/enemy of the corporation. I doubt they would have good faith for a second chance and/or believe in any kind of rehab

**Pull request closed without merging**